package.json

```diff
@@ -19,7 +19,7 @@
   "scripts": {
     "start": "pm2 start .pm2.json",
     "start:debug": "SILEX_DEBUG=true node dist/server/server.js",
-    "build": "npm run build:css && npm run build:html && npm run build:js:release ",
```

Comment:
another solution is to build silex on a CI server (travis or codeship...) and deploy to bluemix the compiled files...